ldap/servers/slapd/daemon.c

```diff
@@ -1123,6 +1160,9 @@ slapd_daemon(daemon_ports_t *ports)
                   "MAINPID=%lu",
                   (unsigned long)getpid());
 #endif
+    slapi_eq_repeat_rel(check_idletimeout, NULL,
+                        slapi_current_rel_time_t(),
+                        MILLISECONDS_PER_SECOND);
```

Comment:
I wonder if it's not too often... (scanning all of the connections every second)

Probably, it makes sense to see that there is no performance drop in situations with many user connections.

Reply:
I doubt we will notice anything: 
   During stress test the listener threads are walking all the active connections every tenth of ms (or even faster)  to build and check the polling table so walking it every second should have an impact below 0.01%